rag_experiment_accelerator/evaluation/tests/test_plain_metrics.py

```diff
@@ -17,7 +17,8 @@ def test_fuzzy():
     value1 = "Room, 2 Double Beds (19th to 25th Floors)"
     value2 = "Two Double Beds - Location Room (19th to 25th Floors)"
 
-    assert fuzzy(value1, value2) == 97
+    assert int(fuzzy(str1=value1, str2=value2)) == 89
+    assert int(fuzzy(str1=value1, str2=value2, match_type="partial_token_set_ratio")) == 100
```

Comment:
Severity: Potential Issue - The explicit cast to `int` might not be necessary if `fuzzy` already returns an integer. If it does not, consider handling the type conversion within the `fuzzy` function itself for better encapsulation.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.

Reply:
actual test here is to validate float rather than int. will switch to comapring upto significant digits

Reply:
done